accelerator/BUILD

```diff
@@ -28,7 +28,6 @@ cc_library(
         ":ta_errors",
         "//serializer",
         "@entangled//common/trinary:trit_tryte",
-        "@entangled//mam/api",
```

Comment:
I couldn't build without this library. And there's nowhere included `mam` library.

Reply:
I will add it back